src/utils/hooks/useMergeRefs.ts

```diff
@@ -0,0 +1,21 @@
+export default function useMergeRefs<T>(
+    ...inputRefs: (React.Ref<T> | undefined)[]
+): React.Ref<T> | React.RefCallback<T> {
+    const filteredInputRefs = inputRefs.filter(Boolean);
```

Comment:
Is this the same as `inputRefs.filter(ref => ref)`?

Reply:
Yes, its functionally the same, Boolean would be in this case kind of a shorthand for filtering out falsy values. I can update it to use `ref => ref` for explicitness if that's preferred. Let me know!

Reply:
I think it's fine the way it is.. thanks!